packages/clerk-js/src/ui/components/Checkout/CheckoutForm.tsx

```diff
@@ -180,10 +180,13 @@ const useCheckoutMutations = () => {
       useTestCard: true,
     });
 
+  const subscribeWithoutPaymentMethod = () => confirmCheckout({});
```

Comment:
Nit: I can see that `ConfirmCheckoutParams` type is a union with inner sets of optional properties, therefore passing an empty object works here 

I'd suggest allowing passing `undefined` here such as `confirmCheckout()` instead